tailwind.config.js

```diff
@@ -0,0 +1,123 @@
+module.exports = {
+  theme: {
+    borderColor: theme => ({
+      1: theme("colors.border.1"),
+      2: theme("colors.border.2"),
+      3: theme("colors.border.3"),
+      4: theme("colors.border.4"),
+      5: theme("colors.border.5"),
+    }),
+    borderWidth: {
+      0: "0",
+      1: "1px",
+      3: "3px"
+    },
+    borderRadius: {
+      none: '0',
+      1: "var(--calcite-border-radius)",
```

Comment:
Extending the conversation [here](https://github.com/Esri/calcite-components/pull/897/files#r477494634) we may want to consider moving [this](https://github.com/Esri/calcite-components/blob/master/src/assets/styles/global.scss#L18) to something like `2px`. I don't think we need to address this now - just wanted to make the note.

Reply:
Sounds good, agree would be nice to hit a scale value. Probably to 4px since 2px as a first value about 0 is pretty imperceptible. Thoughts cc @bstifle

Reply:
> Probably to 4px since 2px as a first value about 0 is pretty imperceptible

I'd say depends on the pixel density of the display. At 72 ppi, `2px` might as well be `0px`. On a scaled, high res display, its much more noticeable, atleast, to my eyes anyway. The future is definitely high res so a `2px` option can be nice to have around.